app/javascript/controllers/search_controller.js

```diff
@@ -0,0 +1,25 @@
+import { Controller } from "@hotwired/stimulus"
+// Connects to data-controller="search"
+export default class extends Controller {
+  static targets = ["params"]
+
+  connect() {
+    const value = this.paramsTarget.value
+    console.log(value)
```

Comment:
z tego mozemy zrezugnować ;) console.log wypluwa rzeczy do konsoli JS - przy testach się to przydaje, ale "produkcyjnei" to mało eleganckie ;)